### DISCUSSION TITLE
Usage precisions

### DISCUSSION THREAD

**Discussion by nbourdeau** [kartoza/docker-geoserver#343] [Q&A]
Hi,

Reading documentation, I can't find how to

1. Change logging profile of docker image to "PRODUCTION_LOGGING"
2. Find documentation on the CSV data store usage ?

Somebody can help ?

Thanks

**Comment by NyakudyaA** [comment]
You can use the env variable `GEOSERVER_LOG_LEVEL=PRODUCTION` as an env variable but remember this can be overwritten by changes made through the gui.

For the csv data storage you need to use the `importer` extension or if they are other native extension for csv
